technic/machines/register/machine_base.lua

```diff
@@ -210,7 +210,7 @@ function technic.register_base_machine(data)
 		after_dig_node = technic.machine_after_dig_node,
 		on_receive_fields = function(pos, formname, fields, sender)
 			local node = minetest.get_node(pos)
-			if not pipeworks.may_configure(pos, sender) then return end
+			if fields.quit or not pipeworks.may_configure(pos, sender) then return end
```

Comment:
Independent checks. I'd prefer to see them separate.
```suggestion
			if fields.quit then return end
			if not pipeworks.may_configure(pos, sender) then return end
```

Please also do this change for the `_active` node.

Reply:
Done.